src/main/java/com/spikes2212/command/genericsubsystem/MotoredGenericSubsystem.java

```diff
@@ -21,8 +22,8 @@ public class MotoredGenericSubsystem extends GenericSubsystem {
      * minSpeed supplier, the given maxSpeed supplier and the given {@link MotorController}s.
```

Comment:
no need to write `the given` each time. also change to `the minimum speed` and `the maximum speed` instead of `minSpeed` and `maxSpeed`